packages/@react-spectrum/meter/stories/Meter.stories.tsx

```diff
@@ -10,139 +10,160 @@
  * governing permissions and limitations under the License.
  */
 
+import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
 import {Meter} from '../';
 import React from 'react';
-import {storiesOf} from '@storybook/react';
+
+type MeterStory = ComponentStoryObj<typeof Meter>;
 
 const formatOptions = {
   style: 'currency',
   currency: 'JPY'
 };
 
-storiesOf('Meter', module)
-  .addParameters({
-    providerSwitcher: {status: 'positive'},
-    args: {value: 32},
-    argTypes: {
-      value: {
-        control: {
-          type: 'range',
-          min: 0,
-          max: 100
-        }
+export default {
+  title: 'Meter',
+  component: Meter,
+  argTypes: {
+    value: {
+      control: {
+        type: 'range',
+        min: 0,
+        max: 100
+      }
+    },
+    variant: {
```

Comment:
with these, we shouldn't need any of the stories that are specific to the variant or size anymore
I believe there is also a size 'M', so that can't be a radio, it should be a select, and then we won't need any stories specific to the size anymore

if there's a specific reason you aren't removing any of the mentioned stories, let me know, happy to discuss it

Reply:
'M' isn't in the types apparently
https://github.com/adobe/react-spectrum/blob/0c241bfeafb3bf22a6764cea5424449b3dd7daaa/packages/%40react-types/progress/src/index.d.ts#L31-L32

Reply:
you are right, got confused between the circle version and spectrum's docs https://spectrum.adobe.com/page/progress-bar/#Size
leave as is for now, our CSS doesn't support medium, I just checked

Reply:
And I can definitely remove a lot of the stories. Just not sure exactly where we want to draw the line.